libs/verify-android-environment.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 if [[ -z "${ANDROID_NDK_ROOT}" ]]; then
   echo "Could not find Android NDK:"
-  echo 'Please install the Android NDK r15c and then set ANDROID_NDK_ROOT.'
+  echo 'Please install Android NDK r20 and then set ANDROID_NDK_ROOT.'
```

Comment:
Should we check that it's r20 somehow? Seems very likely people will miss the memo to update this...

Probably worth sending out an email too, tbh.

Reply:
We actually check a bit further down the file :)

Reply:
Good idea on the email!